wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -797,6 +797,32 @@ The following is an example of what a table should look like:
 | Maria | No Contest | Mocha |  |
 ```
 
+## Infoboxes
+
+An infobox is a fixed-width block which is aligned to the right side of the article. It may contain a relevant image, which explains the surrounding text, or a block of navigation that links to other articles from the same category.
+
+Example use, rendered on the right:
+
+<!-- The real infobox is added for illustrative purposes, with Markdown syntax duplicated below for clarity. -->
+
+::: Infobox
+![](/wiki/shared/mods/SD.png "Sudden Death mod icon")
+:::
+
+```markdown
+::: Infobox
+![](/wiki/shared/mods/SD.png "Sudden Death mod icon")
+:::
+```
+
+The title text serves as the image caption. If an image inside the infobox is wider than 300px<!-- https://github.com/ppy/osu-web/blob/bca69155332cb0c020ba225f2f9de579ad99284c/resources/assets/less/bem/osu-md.less#L226 -->, its thumbnail will be displayed instead, with full version available on click. Images from all infoboxes on the page are combined into a single gallery, which can be navigated using arrow icons on both sides of the screen<!-- https://github.com/ppy/osu-web/pull/8126 -->.
```

Comment:
>The title text serves as the image caption.

that's a general thing - a single image on its own line will have its title text displayed as a caption. infoboxes are like tiny little markdown sections

> Images from all infoboxes on the page are combined into a single gallery, which can be navigated using arrow icons on both sides of the screen

*any* image that is on its own are viewable in the gallery, again this doesn't only apply to infoboxes

(+arrow keys, swiping, etc. also work for navigation)

Reply:
didn't know that. guess the whole paragraph needs to be removed, or partially moved into the images section